src/script.js

```diff
@@ -1,3 +1,37 @@
 "use strict";
 
 // TODO - write your code here.
+
+const randomDamage = () => Math.floor(Math.random() * 10 + 1);
+
+const chooseOption = (opt1, opt2) => {
+    const randNum = Math.round(Math.random());
+
+    return randNum ? opt2 : opt1;
+};
+
+const attackPlayer = function (health) {
+    return health - randomDamage();
+};
+
+const logHealth = (player, health) =>
+    console.log(`${player} health: ${health}`);
+
+const logDeath = (winner, loser) => console.log(`${winner} deafeated ${loser}`);
+
+const isDead = (health) => health <= 0;
+
+function fight(player1, player2, player1Health, player2Health){
```

Comment:
Looks like you're off to a good start on the fight() function. You just need to finish it and add the conditional for when the attacker is player2.